platforms/documentation/docs/src/docs/userguide/best-practices/best_practices_dependencies.adoc

```diff
@@ -162,6 +162,56 @@ include::sample[dir="snippets/bestPractices/settingsRepositoriesDo/groovy",files
 
 `<<tags_reference.adoc#tag:structuring-builds,#structuring-builds>>`
 
+[[use_content_filtering]]
+== Use Content Filtering when using multiple Repositories
+
+When using multiple repositories, use <<filtering_repository_content.adoc#repository-content-filtering,repository content filtering>> to ensure that dependencies are resolved from an appropriate repository.
+
+=== Explanation
+
+If your build declares more than one repository, you should also declare content filters on these repositories to ensure you search for and obtain dependencies from the correct place.
+The repository that contains the remaining artifacts should come at the end, so we only query it if we don’t find the included modules in the other repositories.
+Another option is to use the <<filtering_repository_content.adoc#sec:declaring-content-repositories,exclusiveContent>> if you are able to. Here, the order doesn’t matter.
+
+Content filtering has three main benefits:
+
+- **Performance** since you only query one repository for a given artifact
+- **Security**, by avoiding asking potentially every repository for every dependency (even ones they shouldn't contain), and improving resiliency to supply chain attacks by specifying exactly where to obtain each artifact
+- **Reliability**, by avoiding searching repositories that contain invalid or incorrect metadata or artifacts for particular dependencies
+
+=== Example
+
+==== Don't Do This
+
+Don't add multiple repositories without content filtering:
+
+====
+include::sample[dir="snippets/bestPractices/content-filtering-avoid/kotlin",files="settings.gradle.kts[tags=content-filtering-avoid]"]
+include::sample[dir="snippets/bestPractices/content-filtering-avoid/groovy",files="settings.gradle[tags=content-filtering-avoid]"]
+====
+
+==== Do This Instead
+
+Use exclusive content filtering to ensure that the additional repositories are used only for the expected artifacts:
+
+====
+include::sample[dir="snippets/bestPractices/content-filtering-exclusive-do/kotlin",files="settings.gradle.kts[tags=content-filtering-do]"]
+include::sample[dir="snippets/bestPractices/content-filtering-exclusive-do/groovy",files="settings.gradle[tags=content-filtering-do]"]
+====
+
+Alternatively, using content filtering for additional repositories while having the fallback repository at the end is also a good option.
+
+NOTE: Not using content filtering on the last repository may be a security risk since you might resolve artifacts that you expect to be resolved from other repositories with content filtering from it, so make sure you trust the repository.
```

Comment:
```suggestion
NOTE: If you don’t apply content filtering to the last declared repository, you may inadvertently resolve artifacts from it that were intended to come from earlier, trusted repositories with content filtering. This can pose a security risk, so make sure you fully trust the repository.
```